lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -134,6 +137,17 @@ export function setToken(
     }
 }
 
+function removeTokenUpdatesFromResponse(res: BaseResponse) {
+    res.removeHeader(antiCsrfHeaderKey);
+    res.removeHeader(frontTokenHeaderKey);
+
+    res.removeHeader(getResponseHeaderNameForTokenType("access"));
+    res.removeHeader(getResponseHeaderNameForTokenType("refresh"));
+
+    res.clearCookie(getCookieNameFromTokenType("access"));
+    res.clearCookie(getCookieNameFromTokenType("refresh"));
```

Comment:
do we really need to do this? It's a very weird function to have in the framework.. What happens if we don't do this?

Reply:
The worst case is shown in a test: if the user calls `createNewSession` then throws an UNAUTHORISED it will not clear the new session. This can be a use-case, for example, in a user banning scenario: checking the signed-in user in the `signInPOST` override.